CMakeLists.txt

```diff
@@ -333,6 +333,20 @@ function(generateInstallDirectives)
     )
 
   elseif(PLATFORM_WINDOWS)
+
+    # CMake doesn't prefer 'Program Files' on x64 platforms, more information
+    # here: https://gitlab.kitware.com/cmake/cmake/-/issues/18312
+    # This is a workaround, to ensure that we leverage 'Program Files' when
+    # on a 64 bit system.
+    if(WIN32)
+      if(CMAKE_SIZEOF_VOID_P MATCHES "8")
+          set(CMAKE_INSTALL_PREFIX "Program Files/${CMAKE_PROJECT_NAME}")
```

Comment:
The install prefix is not what controls where the MSI will install. This is the prefix where the `install` command will copy the files, which in the case of CPack will only control where the files are in its internal intermediary folder.

Also, why this is an issue?
Did something change with the latest version? because osquery installs under `C:\Program Files` correctly if the binary is 64bit/